controllers/configmap_controller_test.go

```diff
@@ -146,67 +209,3 @@ func TestParseInstances(t *testing.T) {
 		})
 	}
 }
-
-func TestIsValidConfigMap(t *testing.T) {
```

Comment:
Again, why are you moving this function around? Doing is forcing the reviewer to re-review this function. If you want to mechanically move functions around, open separate PRs.